docs/core/project-sdk/msbuild-props.md

```diff
@@ -1421,6 +1421,116 @@ The `RunWorkingDirectory` property defines the working directory for the applica
 </PropertyGroup>
 ```
 
+## Test project&ndash;related properties
+
+The following MSBuild properties are documented in this section:
```

Comment:
Are we ok with mentioning MSTest specific properties? If so I would also add:

`UseWinUI` to set to true when testing WinUI UI components..
`EnableMSTestV2CopyResources` (default value is `true`) that allows to copy resources dll to the output

Reply:
Tracking in https://github.com/dotnet/docs/issues/42544.